cob_teleop/ros/src/cob_teleop.cpp

```diff
@@ -343,14 +345,19 @@ void CobTeleop::updateBase(){
   }
 }
 
-
-void CobTeleop::joy_cb(const sensor_msgs::Joy::ConstPtr &joy_msg){
-
-  if(mode_switch_button_>=0 && mode_switch_button_<(int)joy_msg->buttons.size() && joy_msg->buttons[mode_switch_button_]==1)
+void CobTeleop::say(std::string text, bool blocking)
+{
+  cob_sound::SayGoal saygoal;
+  std::replace(text.begin(), text.end(), '_', ' ');
```

Comment:
I'd prefer `say` to say what is passed to it.
(Thus I'd prefer to use `replace` where the saytext is composed. Even if that means some duplication)